src/ophys_etl/utils/rois.py

```diff
@@ -349,6 +349,59 @@ def _coo_mask_to_LIMS_compatible_format(coo_mask: coo_matrix
     return compatible_roi
 
 
+def is_in_motion_border(
+        left: int,
+        right: int,
+        top: int,
+        bottom: int,
+        max_correction_right: int,
+        max_correction_left: int,
+        max_correction_up: int,
+        max_correction_down: int,
+        movie_shape: Tuple[int, int]
+        ):
+    """
+    Parameters
+    ----------
+    left: int
+        The left border of the ROI
+    right: int
+        The right border of the ROI
+    top: int
+        The top border of the ROI
+    bottom: int
+        The bottom border of the ROI
+    max_correction_right: int
+        The maximum rightward shift of the movie
+    max_correction_left: int
+        The maximum leftward shift of the movie
+    max_correction_up: int
+        The maximum upward shift of the movie
+    max_correction_down: int
+        The maximum downward shift of the movie
+    movie_shape: Tuple[int, int]
+        The frame shape of the movie from which ROIs were extracted in order
+        of: (height, width).
+
+    Returns
+    -------
+    is_in_border: bool
+        True if the ROI is inside the motion border, False if not.
+    """
+    # A rightward shift increases the min 'valid' left border of the movie
+    l_inset = math.ceil(max_correction_right)
+    # Conversely, a leftward shift reduces the 'valid' right border
+    r_inset = math.floor(movie_shape[1] - max_correction_left)
+    t_inset = math.ceil(max_correction_down)
+    b_inset = math.floor(movie_shape[0] - max_correction_up)
+
+    is_in_border = ((left < l_inset)
+                    | (right > r_inset)
+                    | (top < t_inset)
+                    | (bottom > b_inset))
+    return is_in_border
+
+
 def is_inside_motion_border(
```

Comment:
do we really need both `is_in_motion_border` and `is_inside_motion_border` ? It is just `not` the other. I think having 1 is better.

Reply:
I addressed that here. 

![image](https://github.com/AllenInstitute/ophys_etl_pipelines/assets/10882060/11b039db-ad22-4ef2-993d-cd9bbc1662fe)

It's possible to have a wrapper function that takes an input of an ROI Mask and computes the x, y, width, and height of an ROI and inputs it to create a DenseROI object. But I think it makes sense to implement the simplest quick solution at this point.